app/js/utils/saga.js

```diff
@@ -44,3 +45,22 @@ export function *doTransaction(preSuccess, success, failed, {value = 0, toSend})
     yield put({type: failed, error: error.message});
   }
 }
+
+export function getEnsAddress(name) {
```

Comment:
there is an embark js service, that would probably be a better location

Reply:
We probably need to update the contract to allow non address?

Reply:
Moved. Also, update the contract to be a string instead of an address?